### PR TITLE
fix: CRW-2118: Add httpd and ssl conf security updates

### PR DESCRIPTION
**Pull request by ArvinB**
### What does this PR do?
Updates httpd to deny access to security sensitive files such as .htaccess
Updates ssl to increase encryption strength

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/CRW-2118

#### Release Notes

N/A

### CODE REVIEW

**Comment by ArvinB** [comment]
@nickboldt @ericwill FYI

File: dependencies/che-devfile-registry/build/dockerfiles/Dockerfile [hunk 1]
```diff
@@ -87,6 +88,14 @@ RUN \
     echo "Installed Packages" && rpm -qa | sort -V && echo "End Of Installed Packages"
 
 # BEGIN these steps might not be required
```
Comment (nickboldt) [inline dependencies/che-devfile-registry/build/dockerfiles/Dockerfile:90]: might want to remove this comment as you're adding things that you feel ARE required :)

File: dependencies/che-plugin-registry/build/dockerfiles/Dockerfile [hunk 2]
```diff
@@ -74,6 +75,14 @@ RUN \
     echo "Installed Packages" && rpm -qa | sort -V && echo "End Of Installed Packages"
 
 # BEGIN these steps might not be required
```
Comment (nickboldt) [inline dependencies/che-plugin-registry/build/dockerfiles/Dockerfile:77]: might want to remove this comment as you're adding things that you feel ARE required :)

**Comment by nickboldt** [comment]
Looks good. Two minor nitpicks and a question:

Have you submitted the same fix for che upstream? Any reason why shouldn't also be fixed in upstream?

File: dependencies/che-plugin-registry/build/dockerfiles/Dockerfile (hunk 2 above)
Comment (ArvinB) [reply dependencies/che-plugin-registry/build/dockerfiles/Dockerfile]: Removed.

File: dependencies/che-devfile-registry/build/dockerfiles/Dockerfile (hunk 1 above)
Comment (ArvinB) [reply dependencies/che-devfile-registry/build/dockerfiles/Dockerfile]: Removed.

**Comment by ArvinB** [comment]
> Looks good. Two minor nitpicks and a question:
> 
> Have you submitted the same fix for che upstream? Any reason why shouldn't also be fixed in upstream?

@nickboldt Great question! I didn't really see where to fix this in the upstream of che. Therefore, I went "mid-upstream" for now. :)

**Comment by nickboldt** [comment]
Here's upstream:

https://github.com/eclipse-che/che-devfile-registry/blob/main/build/dockerfiles/Dockerfile
https://github.com/eclipse-che/che-devfile-registry/blob/main/build/dockerfiles/rhel.Dockerfile

https://github.com/eclipse-che/che-plugin-registry/blob/main/build/dockerfiles/Dockerfile
https://github.com/eclipse-che/che-plugin-registry/blob/main/build/dockerfiles/rhel.Dockerfile

**Comment by ArvinB** [comment]
> Here's upstream:
> 
> https://github.com/eclipse-che/che-devfile-registry/blob/main/build/dockerfiles/Dockerfile
> https://github.com/eclipse-che/che-devfile-registry/blob/main/build/dockerfiles/rhel.Dockerfile
> 
> https://github.com/eclipse-che/che-plugin-registry/blob/main/build/dockerfiles/Dockerfile
> https://github.com/eclipse-che/che-plugin-registry/blob/main/build/dockerfiles/rhel.Dockerfile

@nickboldt 
I will work on submitting a PR upstream to those files here today. Thank you.

**Review by svor — APPROVED** [review]

**Comment by nickboldt** [comment]
replaced by https://github.com/redhat-developer/codeready-workspaces/pull/557 so we can run tests that require secrets (must be PR from origin fork, not user fork).

**Pull request closed without merging**